doc/usermanual/darkroom/modules/tone/filmicrgb.xml

```diff
@@ -0,0 +1,619 @@
+                <xref linkend="filmic-rgb-for-26-users"/> useful.
+              </para>
+            </entry>
+            <entry>
+              <graphic fileref="darkroom/modules/images/filmic.png" scalefit="1" width="80%"
+                align="center"/>
+            </entry>
+          </row>
+        </tbody>
+      </tgroup>
+    </informaltable>
+
+  </sect4>
+
+  <sect4>
+
+    <title>Prerequisites</title>
+
+<!-- IHU - my understanding is that the prerequisites don't change much, except for the 
+      exposure settings - which I modified. Is that correct? -->
+
+    <para>
+      In order to get the best out of filmic rgb, images need some preparation:
+      <itemizedlist>
+
+        <listitem><para>
+          In-camera, expose the shot <quote>to the right</quote>. This implies under exposing
+          the shot so that the highlights are at the right of the histogram, just on the verge
+          of clipping, but not clipped. It does not matter if the picture preview is very dark
+          on your camera screen: as long as highlights are unclipped, filmic rgb should be able
+          to recover details from the raw data. Beware that clipped data are not recoverable.
+          Some cameras have a clipping alert preview to help you diagnose this, and some even
+          have an highlight-priority exposure mode.
+        </para></listitem>
+
+        <listitem><para>
+          In the <link linkend="exposure"><emphasis>exposure</emphasis></link> module, push the
+          exposure until the midtones are clear enough. Do not worry about losing the
+          highlights: they will be recovered as part of the filmic rgb processing. However, it
+          is important to avoid negative pixels in black areas: the computations done by filmic
+          rgb will result in unpredictable results in this case. For some cameras models (Canon,
+          mainly), rawspeed, the raw decoding library of darktable, may set an exaggerated black
+          level, resulting in crushed blacks and negative values. If so, brighten the blacks by
+          setting a negative black level value in the
+          <link
+              linkend="exposure"><emphasis>exposure </emphasis></link> module.
+        </para></listitem>
+
+        <listitem><para>
+          If you plan on using filmic rgb's auto-tuners, use the
+          <link
+              linkend="whitebalance"><emphasis>white balance</emphasis></link>
+          module first to correct any color casts and get neutral colors. In RGB color spaces,
+          luminance and chrominance are linked, and filmic rgb's luminance detection relies on
+          accurate measures of both. If your picture is very noisy, add an initial step of
+          denoising to help the black exposure readings, and use a high quality demosaicing.
+        </para></listitem>
+
+        <listitem><para>
+          If you plan using filmic rgb's chrominance preservation mode, avoid using any tone
```

Comment:
If you plan on using filmic rgb's chrominance preservation mode, avoid using any tone